README.md

```
@@ -72,28 +73,39 @@ NEXT_PUBLIC_FORMSPREE_FORM_ID=
+
 ```sh
-docker run -p 3000:3000 --env-file .env my-portfolio-app
+docker run -p 3000:3000 --env-file .env.local my-portfolio-app
 ```
 
-- The app will be available at [http://localhost:3000](http://localhost:3000)
-- Make sure to set up your `.env` file with the required environment variables before running.
+The app will be available at [http://localhost:3000](http://localhost:3000)
+Make sure to set up your `.env.local` file before running.
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)